schemas/rdf/examples/generated/AdministrativeInformation/maximal.ttl

```diff
@@ -5,7 +5,6 @@
 @prefix xs: <http://www.w3.org/2001/XMLSchema#> .
 
 <something_142922d6> rdf:type aas:AssetAdministrationShell ;
-    <https://admin-shell.io/aas/3/0/Referable/idShort> "something_783819f1"^^xs:string ;
```

Comment:
It looks like that the `idShort` property disappeared in all the examples. Can we get them back?

Reply:
No, that is unfortunately not possible as ID-shorts are optional properties. Their presence in the previous examples was due to the bug I introduced when I misinterpreted that identifiables *must* have ID-shorts, while in reality they do not.

For some history & future reference, see the following pull requests:
* https://github.com/aas-core-works/aas-core-codegen/pull/365: I introduced the type strengthening in aas-core-codegen after introducing the constraint in aas-core-meta with https://github.com/aas-core-works/aas-core-meta/pull/256/files and https://github.com/aas-core-works/aas-core-meta/pull/261.
* After realizing that I misinterpreted Constraint AASd-117, I reverted the changes in https://github.com/aas-core-works/aas-core-meta/pull/264 and https://github.com/aas-core-works/aas-core-codegen/pull/379.

The logic for `maximal` tests in testgen sets all the required properties of the relevant object (an instance of `AdministrativeInformation` in your example), but embeds it in a minimal environment for readability. When I erroneously switched `IdShort` to required in https://github.com/aas-core-works/aas-core-meta/pull/261, testgen obviously included `IdShort` in the minimal examples of the wrapping objects around the instance of `AdministrativeInformation`.

Now that I reverted back to the (correct) specification in https://github.com/aas-core-works/aas-core-meta/pull/264, testgen generates the minimal wrapping without `IdShort` as before.

Reply:
> After realizing that I misinterpreted Constraint AASd-117, I reverted the changes in

I see, the formulation of `AASd-117` and its note is in Part 1 V3.0 a little bit different than I remembered it. The new state is therefore indeed correct.